### ISSUE TITLE
Error handling

### ISSUE THREAD

**Issue by jbellik** [syntax-prosody-ot/main#85]
Currently, all Errors only display to the console, even when they pertain to data issues. These should be Alerts as well -- or something prettier.

**Comment by jbellik** [comment]
This also entails figuring out where errors are being thrown...

**Comment by jbellik** [comment]
Start by checking interface1.js.

**Comment by ivpetkov** [comment]
I added alerts for a few errors in the error-handling branch. Some files had errors displayed in the console that I think aren't important to users, but I wasn't sure about the files mentioned below.

- interface.js: The errors that result from the confirms don’t display on the interface, but do they need to? All other errors that concern the user are either alerts or confirms.
- built-in_analyses.js: Added alert
- candidategenerator.js: Are these errors important to user?
- noShift.js: Added alert
- inputCandidateGenerator.js: Are these errors important to user?
- prosodicHierarchy.js: Are these errors important to user?
